app/components/chart-with-filters.tsx

```diff
@@ -165,7 +165,8 @@ export const ChartWithFilters = React.forwardRef<
   useSyncInteractiveFilters(props.chartConfig);
 
   return (
-    <div ref={ref}>
+    // Minimum aspect ratio for any chart (0.4)
+    <div ref={ref} style={{ width: "100%", aspectRatio: "5 / 2" }}>
```

Comment:
Not really an idea, but maybe the debugger can be useful to investigate. Below you'll see that I captured a trace of what was happening and after I get access to screenshots of whats' happening and we can see that there is a moment when nothing is displayed, so that must be the culprit :) I am pretty sure that chrome has the same kind of tool.

https://github.com/visualize-admin/visualization-tool/assets/465582/09289150-9073-4515-856d-31e1bc79f1c9